smartlogic/client.py

```diff
@@ -0,0 +1,86 @@
+from django.conf import settings
+
+try:
+    assert settings.SMART_LOGIC_ENVIRONMENT
+except AssertionError:
+    from django.core.exceptions import ImproperlyConfigured
+
+    raise ImproperlyConfigured(
+        "Please provide a value for SMART_LOGIC_ENVIRONMENT in settings.py"
+    )
+
+import json
+
+import requests
+from requests.exceptions import HTTPError, ConnectTimeout
+
+from smartlogic.exceptions import (
+    RequestFailed,
+    RequestNotAuthenticated,
+    AuthenticationFailed,
+    ResponseNotSerializable,
+)
+
+
+class SmartLogic(object):
+    BASE_URL = "https://cloud.smartlogic.com"
+    SERVICE_URL = f"/svc/{settings.SMART_LOGIC_ENVIRONMENT}/ses/CombinedModel"
+
+    def __init__(self, api_key, authorization=None):
+        self.api_key = api_key
+        self._authorization = authorization
```

Comment:
Authorization needs to be provided by views using this client, which is great because it means you can't just request our smartlogic endpoints willy nilly.